ecs-cli/modules/cli/local/project/project.go

```diff
@@ -175,30 +163,18 @@ func (p *localProject) Convert() error {
 	// FIXME get secrets here, pass to converter?
 	// NOTE: Should add log message to warn user that decrypted secret
 	// will be written to local compose file
-	services, err := converter.ConvertToDockerCompose(p.taskDefinition)
 
-	if err != nil {
-		return err
-	}
+	var (
+		data []byte
+		err  error
+	)
 
 	if Arn != "" {
```

Comment:
We should avoid using global variables whenever we can because it makes unit testing very difficult. In this situation we can avoid it because of `p.context`.